### DISCUSSION TITLE
Which dark mode, if any, should style the editor join flow?

### DISCUSSION THREAD

**Discussion by mxmou** [ScratchAddons/ScratchAddons#3313] [Discussion]
Website dark mode uses `:not(.sa-body-editor)` selectors to make sure that it doesn't affect the editor join flow, and https://github.com/ScratchAddons/ScratchAddons/pull/1805#issuecomment-796222183 suggests that editor dark mode isn't meant to affect it either. I don't think it makes sense for it to remain light when both website and editor dark mode are enabled.

**Comment by CST1229** [comment]
What is the editor join flow?
EDIT: Is it the join scratch modal in the editor?

**Reply by mxmou** [reply]
Yes, it's the join modal in the editor.

**Comment by WorldLanguages** [comment]
IMO, none is fine, but if you really want to, style it through editor dark mode.
By `This does not affect "join scratch" modal, epic!`, I was specifically referring to some dark modes that styled all `input`s, instead of using specific class names, so the join flow would look weird, and sometimes text would be unreadable AFAIK. I was not really trying to point out that it should be styled by website dark mode.
